README.rst

```diff
@@ -76,6 +76,10 @@ Setup redis connection and enable caching for desired models:
     # should be compatible or subclass cacheops.redis.CacheopsRedis
     CACHEOPS_CLIENT_CLASS = 'your.redis.ClientClass'
 
+    # To use your own serializer class,
+    # should be compatible or subclass cacheops.serializers.PickleSerializer
+    CACHEOPS_SERIALIZER = 'your.serializers.ClientClass'
```

Comment:
It's much easier to simply point to a module. And default will be simply `pickle`.